.scripts/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+set -euox pipefail
```

Comment:
Does this work? `pipefail` is connected to the `-o` option only. My bash isn't great, but I'm sure `set -euxo pipefail` works, not sure about this version

Reply:
I think I need to create any build failure to verify it. in travis